# This service is used to delete an existing image thumbnail from an existing
# object on the interoperability server.

# Object ID.
uint64 id

---

# Whether the request was successful.
bool success
